src/tooling/docs-assembler/Navigation/GlobalNavigationHtmlWriter.cs

```diff
@@ -32,13 +32,25 @@ private bool TryGetNavigationRoot(
 		navigationRootSource = null;
 		if (!assembleSources.TocTopLevelMappings.TryGetValue(navigationSource, out var topLevelMapping))
```

Comment:
Long term all of this will go away and we can just query the navigation itself..